### DISCUSSION TITLE
MapView vs. FirstPersonView: difference in the display of circles.

### DISCUSSION THREAD

**Discussion by zsoooc** [visgl/deck.gl#8212] [Q&A]
Hello!
I would like to use FirstPersonView to display points on the map (like a GeoJSON layer). The data source is MVTLayer. The problem is that the circles appear blurry. In addition to this, the "billboard: true" option does not work. So, I tried to use "renderSublayers". 
With using this option, displaying of the circles is perfect if I use the default view (MapView). But, if I use FirstPersonView, the result is blurried and the circles are "flat" (the same as without "renderSubLayers).

Is there any solution to display the circles in FirstPersonView as in MapView?

This is the blurried circle (FirstPersonView):
   
![image](https://github.com/visgl/deck.gl/assets/19270270/30407a19-b089-4eff-bb96-3bb8e495081e)

This one is the same object, but with the MapView:
![image](https://github.com/visgl/deck.gl/assets/19270270/a73bdc44-7dcf-4d5e-bae4-ebd285f3368e)

My code:
The layer:

```
new deck.MVTLayer({
    data: 'http://127.0.0.1:7800/layer/{z}/{x}/{y}.pbf?',
    id: 'places',
    renderSubLayers: props => {
        return new deck.GeoJsonLayer(props, {
            pointType: 'circle',
            radiusMinPixels: 20,
            radiusMaxPixels: 80,
            getPosition: d => d.geometry.coordinates,
            getFillColor: d => [66, 135, 245],
            getPointRadius: 10,
            billboard: true
        });
    }
});
```

The DeckGL object (default MapView):
```

let deckgl = new deck.DeckGL({
    initialViewState: {
        longitude: 16,
        latitude: 45,
        zoom: 20,
        maxZoom: 23,
        pitch: 90,
        bearing: 0
    },
    controller: true,
    layers: layers2
});
```
The DeckGL object (First person View):

```
let deckgl = new deck.DeckGL({
    views: new deck.FirstPersonView(),
    initialViewState: {
        longitude: 16,
        latitude: 45,
        position: [0, 0, 20],
        pitch: 0,
        bearing: 0
    },
    controller: true,
    layers: layers2
});
```

**Comment by Pessimistress** [comment]
There is no prop called `billboard` in GeoJsonLayer/MVTLayer. https://deck.gl/docs/api-reference/layers/geojson-layer

**Reply by zsoooc** [reply]
Ok, thanks for this information, this is not problem for me, I can live without that. But the blurred display of the circle is disturbing. Is it possible to make the circle look nice in First Person View?

**Reply by Pessimistress** [reply]
You can try `pointAntialiasing: false`.

**Reply by Pessimistress** [reply]
> I can live without that

Look for `billboard` in the linked doc page please. You need to specify what geometry you want billboarded.
